addons/website_event/views/event_event_views.xml

```diff
@@ -29,9 +29,11 @@
                 <attribute name="context">{'default_website_id': website_id}</attribute>
                 <attribute name="groups">website.group_multi_website</attribute>
             </xpath>
+            <xpath expr="//div[@name='button_box']/button[hasclass('oe_stat_button')]" position="before">
```

Comment:
Does not seem super robust, can't we instead properly target the button? Probably by using its action?

Reply:
Did so to ensure that the button always stays at the first position even if any other buttons are added before the `Registration` button in future. 

Anyways, used the action of `Registration` button for xpath to place it at the first position as of now :+1: